src/main/python/image_processing/sample_extract.py

```diff
@@ -25,6 +38,9 @@ def extract(self, img):
             id_details: JSON obj (The extracted information)
         Todo:
         """
+        if 'remove_face' in self.preferences:
+            remove_face = self.preferences['remove_face'] == 'true'
```

Comment:
Yes, but in the python side, is the value received an actual string value or a boolean? As I understand it, Python automatically converts any boolean values received by an AJAX request to the corresponding boolean values in Python.